docs/4.0/getting-started/options.md

```diff
@@ -8,13 +8,15 @@ toc: true
 
 ## Customizing variables
 
-Every Sass variable in Bootstrap 4 includes the `!default` flag, meaning you can override that default value in your own Sass even after that original variable's been defined. Copy and paste variables as needed, modify the values, remove the `!default` flag, and recompile.
-
-For example, to change out the `background-color` and `color` for the `<body>`, you'd do the following:
+Every Sass variable in Bootstrap 4 includes the `!default` flag, meaning you can override that default value in your own Sass. Copy and paste variables as needed, modify the values and remove the !default flag. If a variable has already been assigned, then it won't be re-assigned by the default values in Bootstrap. This means that your modified Sass variables should be called before you import Bootstrap Sass files. For example, to change out the `background-color` and `color` for the `<body>`, you'd do the following::
 
 {% highlight scss %}
-$body-bg:    $gray-900;
-$body-color: $gray-600;
+// Your variable overwrite first or a Sass file containing the modifications
+$body-bg: #000000;
```

Comment:
I'd change this to `#000` and `#111` just to follow bootstrap [coding guidelines](http://codeguide.co/)

Reply:
@andresgalante You're right, did that with the new commit